sql/core/src/test/scala/org/apache/spark/sql/execution/metric/SQLMetricsSuite.scala

```diff
@@ -182,10 +182,13 @@ class SQLMetricsSuite extends SparkFunSuite with SQLMetricsTestUtils with Shared
   }
 
   test("Sort metrics") {
-    // Assume the execution plan is
-    // WholeStageCodegen(nodeId = 0, Range(nodeId = 2) -> Sort(nodeId = 1))
-    val ds = spark.range(10).sort('id)
-    testSparkPlanMetrics(ds.toDF(), 2, Map.empty)
+    // Assume the execution plan with node id is
+    // Sort(nodeId = 0)
+    //   Exchange(nodeId = 1)
+    //     Range(nodeId = 2)
+    val df = spark.range(9, -1, -1).sort('id).toDF()
+    testSparkPlanMetrics(df, 2, Map.empty)
+    df.queryExecution.executedPlan.find(_.isInstanceOf[SortExec]).getOrElse(assert(false))
```

Comment:
`assert(df.queryExecution.executedPlan.find(_.isInstanceOf[SortExec]).isDefined)`?

Reply:
OK, I think this is more readable. fixed.